packages/terra-popup/src/Popup.jsx

```diff
@@ -196,6 +211,14 @@ class Popup extends React.Component {
     this.arrowNode = node;
   }
 
+  addInertAttribute = () => {
+    document.querySelector('#root').setAttribute('inert', 'true');
+  };
+
+  removeInertAttribute = () => {
+    document.querySelector('#root').removeAttribute('inert');
+  };
```

Comment:
Why are we adding the inert attribute?  This is not supported on all required browsers such as IE 11.  In addition, it does not seem correct to add to the root level.